translationMigrations/000053-1760619346791.json

```diff
@@ -1,11 +1,11 @@
 {
-    "parent": "000051-1758354992698.json",
```

Comment:
Lets remove translation migration for now. We can do that all together in the future.